example-2/evm/src/contracts/lendingSpoke/SpokeState.sol

```diff
@@ -1,7 +1,9 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.0;
 
-contract SpokeStorage {
+import "../lendingHub/HubStructs.sol";
+
+contract SpokeStorage is HubStructs {
```

Comment:
why SpokeStorage instead of SpokeState?

Reply:
that's a great question but I was just following the convention in Hub:
`contract HubStorage is HubStructs {
    struct Provider {
        uint16 chainId;
        address payable wormhole;
        address tokenBridge;
        address pyth;
    }

    struct State {
        Provider provider;
        `